common/src/main/kotlin/io/homeassistant/companion/android/common/data/websocket/impl/WebSocketCoreImpl.kt

```diff
@@ -395,10 +425,11 @@ internal class WebSocketCoreImpl(
             if (it.eventFlow == null) {
                 activeMessages.remove(id)
             }
-        }
+        } ?: { Timber.w("Response for message not in activeMessage id($id) skipping") }
     }
 
     private suspend fun handleEvent(response: SocketResponse) {
+        // TODO this probably should be in a separate class to be properly tested
```

Comment:
I decided to not test this part for now since it's mostly routing messages to the proper parser.